db/warehouse/migrate/20230613190449_adjust_external_ids_uniq_idx.rb

```diff
@@ -0,0 +1,14 @@
+class AdjustExternalIdsUniqIdx < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
```

Comment:
I've had issues deploying migrations that do this, @eanders it looked like you encountered it on another staging deployment. How should we handle it?

Reply:
I guess we'll just try it again, we know how to work around it if it times out again

Reply:
Yeah, sorry, this is also on my list to look at.  It _should_ be totally fine to disable the transaction, it jus means it won't lock the table for reading/writing.